asapdiscovery-dataviz/asapdiscovery/dataviz/_html_blocks.py

```diff
@@ -0,0 +1,200 @@
+# Description: HTML blocks for the visualisations in asapdiscovery/dataviz/html_vis.py
+
+
+visualisation_header = """\
+<div id="viewport"
+role="NGL"
+data-proteins='[{
+   "type": "data", "value": "mike_combined", "isVariable": true, "loadFx": "loadmike_combined"
+}]'
+data-backgroundcolor="white"></div>
+<script type="text/javascript">document.getElementById("blue").style.color = "blue";</script>
+<script src="https://code.jquery.com/jquery-3.4.1.min.js"
+              integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
+              crossorigin="anonymous" type="text/javascript"></script>
+<script src="https://unpkg.com/ngl@2.0.0-dev.34/dist/ngl.js" type="text/javascript"></script>
+<script src="https://michelanglo.sgc.ox.ac.uk/michelanglo.js" type="text/javascript"></script>
+"""
+
+
+def _indent(indent_me: str) -> str:
+    return indent_me.replace("\n", "\n" + "    ")
+
+
+def _vis_core(pdb_body: str) -> str:
+    core = f"""<script type="text/javascript">
+    var mike_combined =     `{_indent(pdb_body)}
+        `.replace(/^ +/gm, '');
+
+</script>
+
+
+"""
+    return core
+
+
+visualisation_tail = """\
+<script type="text/javascript">
```

Comment:
Consider putting these HTML and javascript literals into separate files. Would make it easier to edit them in the future with a text editor (e.g. syntax highlighting). Then this module just reads the contents in, or you can get rid of the constants entirely and make a function that grabs the file contents by name.

Doesn't have to be this PR.

Reply:
raised an issue #288